### PR TITLE
CIRC-175 Add copyNumbers to Request.item. Without my refactoring.

### PR DESCRIPTION
**Pull request by Anatolii-Starkov**
…to Requests

Exposed Request.item.copyNumbers field
Updated request's json example
Added tests

### CODE REVIEW

File: ramls/circulation.raml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Circulation
-version: v7.1
+version: v7.2
```
Comment (marcjohnson-kint) [inline ramls/circulation.raml]: This now needs updating to 7.3 as master has already moved on to 7.2 due to CIRC-196. Please merge master at the same time.

File: src/test/java/api/requests/RequestsAPIRelatedRecordsTests.java [hunk 2]
```diff
@@ -101,38 +108,43 @@ public void holdingAndInstanceIdComesFromMultipleRecordsForMultipleRequests()
 
     List<JsonObject> fetchedRequestsResponse = requestsClient.getAll();
 
-    JsonObject firstFetchedRequest = getRecordById(
-      fetchedRequestsResponse, firstRequestId).get();
-
-    JsonObject secondFetchedRequest = getRecordById(
-      fetchedRequestsResponse, secondRequestId).get();
+    JsonObject firstItem = getRecordById(fetchedRequestsResponse, firstRequestId).
```
Comment (marcjohnson-kint) [inline src/test/java/api/requests/RequestsAPIRelatedRecordsTests.java]: Please rename this test to better fit the increased scope of it to cover copy numbers

File: ramls/circulation.raml (hunk 1 above)
Comment (Anatolii-Starkov) [reply ramls/circulation.raml]: Done

File: src/main/java/org/folio/circulation/domain/Item.java [hunk 3]
```diff
@@ -156,6 +156,10 @@ public JsonObject getMaterialType() {
     return materialTypeRepresentation;
   }
 
+  public JsonArray getCopyNumbers() {
```
Comment (marcjohnson-kint) [inline src/main/java/org/folio/circulation/domain/Item.java]: Personally, I would convert this to a collection and then map it back to JSON in the representation, however I don't think that is worthy of blocking this pull request.

**Comment by marcjohnson-kint** [comment]
@Anatolii-Starkov Please can master be merged, in order to allow the merge build (above) to complete.

**Pull request merged**